cmd/shell.go

```diff
@@ -18,16 +20,16 @@ import (
 	"syscall"
 	"time"
 
-	"google.golang.org/grpc/grpclog"
-
 	"github.com/go-redis/redis/v8"
 	"github.com/go-sql-driver/mysql"
+	"github.com/letsencrypt/boulder/core"
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/strictyaml"
+	"github.com/letsencrypt/validator/v10"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/collectors"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
-
-	"github.com/letsencrypt/boulder/core"
-	blog "github.com/letsencrypt/boulder/log"
+	"google.golang.org/grpc/grpclog"
```

Comment:
Looks like these imports got accidentally reordered. We keep them in three separate blocks: stdlib packages, packages provided by other repos, and finally packages provided by this repo. Looks like the second and third blocks are comingled here now.